### ISSUE TITLE
Enhancements

### ISSUE THREAD

**Issue by bonmaklad** [butor/blackbird#275]
Hi All, 

I am looking to pull this into my variables. My first task is to get the currencies to be changeable by the config file because USD is now becoming so hard to trade. 

1) As a user I want to change LEG1 and LEG2 AND if applicable the exchanges will send back those trading pairs info SO that I can look to trade

once this is one I want to look not just on bittrex as plenty of others allow short selling. 

2) as a user I want to ensure that IF short selling is allowed for LEG1 and LEG2 then run through all variations to ensure all opportunities are met 

Once complete, I want to do MORE THAN 1 Lens. I want all...

3) As a user I want to cross reference ALL trading pairs to look for the best opportunity SO that I can close positions in less profitable trades. 

Basically.... I want ALL trades possible at any given time to be able to pick up that if this is 0.8% and we are currently at 0.6%... .it will close and take profit and move on. 

This will take me a while. Be best to have some C++ coders with me :)

**Comment by bertrandfalguiere** [comment]
Hi @bonmaklad .

I  was planning to make some change too. I can give you a hand. I'm not a professional dev, but I know some basic C++ and I'm willing to help.
